thumbor/app.py

```diff
@@ -26,22 +22,13 @@ def __init__(self, context):
         super(ThumborServiceApp, self).__init__(self.get_handlers(), debug=self.debug)
 
     def get_handlers(self):
-        handlers = [
-            (self.context.config.HEALTHCHECK_ROUTE, HealthcheckHandler),
-        ]
-
-        if self.context.config.UPLOAD_ENABLED:
-            # Handler to upload images (POST).
-            handlers.append((r"/image", ImageUploadHandler, {"context": self.context}))
-
-            # Handler to retrieve or modify existing images  (GET, PUT, DELETE)
-            handlers.append(
-                (r"/image/(.*)", ImageResourceHandler, {"context": self.context},)
-            )
-
-        if self.context.config.USE_BLACKLIST:
-            handlers.append(
-                (r"/blacklist", BlacklistHandler, {"context": self.context})
+        handlers = []
+        for router in self.context.modules.importer.routers:
```

Comment:
Name `router`/`routers` seems weird. I think name `handlers` has better fit. After all you're putting routes into handlers.

Reply:
I just picked the name after tornado's routing namespace...

Reply:
yes, but after very brief look into tornado documentation, it appears their routers are actually routing - finding appropriate handler. In thumbor case that's not what's happening. Thumbor always uses RuleRouter.